R/pkg/R/context.R

```diff
@@ -435,3 +435,31 @@ setCheckpointDir <- function(directory) {
   sc <- getSparkContext()
   invisible(callJMethod(sc, "setCheckpointDir", suppressWarnings(normalizePath(directory))))
 }
+
+#' Total number of CPU cores of all executors registered in the cluster at the moment.
```

Comment:
`The number reflects current status of the cluster and can change in the future.` this info looks missing here

Reply:
btw, `in this cluster` do we really mean cores allocated to the "application" or "job"? it's not really in the cluster right? If I'm running this app on Hadoop/YARN with 1000s of core, but only set aside 100 for this app, which number am I getting from this API?

Reply:
Hm, yea. I think that's a valid point. It will be 100 in this case.

Reply:
> I think we should really test this.

@HyukjinKwon Unfortunatelly I dont have an opportunity for testing the changes on real Yarn cluster (we don't use it in our company). I can try to create small one on my laptop but I am not sure this is real testing.

Reply:
It's okay for this PR itself for now. I can test.